### PR TITLE
COMMERCE-1553 fix commerce price list tests

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:stop

**Comment by ethanbustad** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
Forgot to update private branch :'(

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

File: commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/internal/search/test/CommercePriceListIndexerTest.java [hunk 1]
```diff
@@ -132,6 +138,8 @@ public void testSearch() throws Exception {
 	private CommercePriceListLocalService _commercePriceListLocalService;
 
 	@DeleteAfterTestRun
+	private Company _company;
+
 	private Group _group;
```
Comment (ethanbustad) [inline commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/internal/search/test/CommercePriceListIndexerTest.java:143]: @luca-pellizzon don't we still want to delete `_group` after the test run?

File: commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/service/test/CommercePriceEntryLocalServiceTest.java [hunk 2]
```diff
@@ -709,13 +714,14 @@ private static void _assertPriceEntryAttributes(
...
-	@DeleteAfterTestRun
 	private CommerceCurrency _commerceCurrency;
 
 	@Inject
 	private CommercePriceEntryLocalService _commercePriceEntryLocalService;
 
 	@DeleteAfterTestRun
+	private Company _company;
+
 	private Group _group;
```
Comment (ethanbustad) [inline commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/service/test/CommercePriceEntryLocalServiceTest.java:725]: (same question here)

File: commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/service/test/CommercePriceListLocalServiceTest.java [hunk 3]
```diff
@@ -642,6 +730,8 @@ private static Date _truncateSeconds(Date date) {
 	private CommercePriceListLocalService _commercePriceListLocalService;
 
 	@DeleteAfterTestRun
+	private Company _company;
+
 	private Group _group;
```
Comment (ethanbustad) [inline commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/service/test/CommercePriceListLocalServiceTest.java:735]: (same question here)

File: commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/service/test/CommerceTierPriceEntryLocalServiceTest.java [hunk 4]
```diff
@@ -434,9 +444,9 @@ private static void _assertTierPriceEntryAttributes(
 		_commerceTierPriceEntryLocalService;
 
 	@DeleteAfterTestRun
-	private Group _group;
+	private Company _company;
 
-	@DeleteAfterTestRun
+	private Group _group;
```
Comment (ethanbustad) [inline commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/service/test/CommerceTierPriceEntryLocalServiceTest.java:449]: (same question here)

File: commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/test/util/CommerceTierPriceEntryTestUtil.java [hunk 5]
```diff
@@ -77,7 +76,8 @@ public static CommerceTierPriceEntry upsertCommerceTierPriceEntry(
 	}
 
 	private static long _getGroupId(
-			Long commercePriceEntryId, String priceEntryExternalReferenceCode)
+			long companyId, Long commercePriceEntryId,
```
Comment (ethanbustad) [inline commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/test/util/CommerceTierPriceEntryTestUtil.java:79]: `Long commercePriceEntryId` should be a primitive `long`, or a NPE would be thrown at `if (commercePriceEntryId > 0)` if the value was `null`

File: commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/internal/search/test/CommercePriceListIndexerTest.java (hunk 1 above)
Comment (luca-pellizzon) [reply commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/internal/search/test/CommercePriceListIndexerTest.java:143]: we are removing the whole company

File: commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/test/util/CommerceTierPriceEntryTestUtil.java (hunk 5 above)
Comment (luca-pellizzon) [reply commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/test/util/CommerceTierPriceEntryTestUtil.java:79]: See https://github.com/liferay/com-liferay-commerce/pull/1414

**Pull request merged**